tests/unit/runtimes/node/in_source_config.test.ts

```diff
@@ -317,5 +336,26 @@ describe('V2 API', () => {
         },
       ])
     })
+
+    test('Using multiple paths', () => {
+      const source = `export default async () => {
+        return new Response("Hello!")
+      }
+  
+      export const config = {
+        path: ["/store/:category/products/:product-id", "/super-awesome-campaign"]
```

Comment:
can we make the second one a named group as well?

+ Can we add a test for what if both routes are the same?
+ What if I add falsy values like `null` or `undefined` to the array? Do we throw type errors if numbers or whatever is provided? (You have a non nullable function but no test cases for it)

Reply:
I've added more tests to capture this.